### DISCUSSION TITLE
Unable to bind port 53 to docker container

### DISCUSSION THREAD

**Discussion by JacobReynolds** [community/community#8703] [Codespaces]
When using the default codespaces image, starting a docker container with port 53 for DNS fails with:
```
$ docker run -p 53:53 -it ubuntu /bin/bash
docker: Error response from daemon: driver failed programming external connectivity on endpoint cool_wing (ba6...): Error starting userland proxy: listen tcp4 0.0.0.0:53: bind: address already in use.
ERRO[0000] error waiting for container: context canceled 
```
Is there a service running that is already binded to this port? I've not been able to find any docker containers claiming this port and no system processes seem to be listening on that port.

**Comment by Geremia** [comment]
`address already in use` means some process is already listening on port 53. Run `lsof -i :53` to check.
